.ignore

```diff
@@ -0,0 +1 @@
+patches/
```

Comment:
?

Reply:
Ignore for `rg` or `ag`, just commented about that.

Reply:
but usually, you'll search in the packages folder? 👀